### PR TITLE
Update script.js

### PR DESCRIPTION
**Pull request by kavjik**
Added some options

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -4,17 +4,21 @@ $(function () {
     "Ukraine",
     "Rusland",
     "Krig",
+    "usikker",
```
Comment (erik-krogh) [inline script.js]:
```suggestion
    "Usikker",
```

File: script.js [hunk 2]
```diff
@@ -4,17 +4,21 @@ $(function () {
...
+    "Usikker",
     "Prins Joachim",
     "Konprinsesse Mary",
+    "Konprinsen",
     "Prins Christian",
     "Grønland",
     "Søens folk",
     "Klima",
     "Færøerne",
+    "Grønland",
```
Comment (erik-krogh) [inline script.js]:
```suggestion
```
duplicate

File: script.js [hunk 3]
```diff
@@ -4,8 +4,10 @@ $(function () {
     "Ukraine",
     "Rusland",
     "Krig",
+    "Usikker",
     "Prins Joachim",
     "Konprinsesse Mary",
+    "Konprinsen",
```
Comment (erik-krogh) [inline script.js]:
```suggestion
    "Kronprinsesse Mary",
    "Kronprinsen",
```
typo

File: script.js (hunk 1 above)
Comment (kavjik) [inline script.js]:
```suggestion
    "Usikker/Usikre",
```

**Pull request merged**